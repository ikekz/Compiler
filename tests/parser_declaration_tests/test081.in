type
	t1 = array of integer;
	t2 = record
			t1: t1;
		end;
begin
end.