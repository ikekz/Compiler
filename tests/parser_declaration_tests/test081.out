integer        type           integer
double         type           double
char           type           char
t1             type           array of integer
t2             type           record:
                              t1             recordfield    array of integer
                              end
